frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -93,7 +93,7 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
         {t('ceph-storage-plugin~Selected Capacity')}
       </div>
       <div className="ceph-ocs-install__stats">
-        <div>
+        <div className="ceph-ocs-install__stats--text-align">
```

Comment:
If this needs to be set for each `div` then cant setting on `ceph-ocs-install__stats"` would work ?

Reply:
yes... you are right... `ceph-ocs-install__stats--text-align` was an unnecessary class, I have removed it...